### DISCUSSION TITLE
v0.6.2024.40-alpha3

### DISCUSSION THREAD

**Discussion by JHRobotics** [JHRobotics/softgpu#107] [Announcements]
**Stability release**

- **! stability fixes**
- **! double buffering support (or emulation)**
- **! Hires <-> VGA mode switching correctly**
- Gamma support (disabled by default)
- Driver: HW cursor in 8/16/32 bpp (VMware, [VirtualBox has this bug](https://www.virtualbox.org/ticket/22096))
- Increased compatibility list

<details>
<summary>checksum</summary>

*MD5:*
```
softgpu-0.6.2024.40.iso: 3de9eff45cbfce68f2940851efe7f447
softgpu-0.6.2024.40.zip: 849aa8d178ba2081ddaf313a8175c5e0
```

*SHA-256:*
```
softgpu-0.6.2024.40.iso: a389071946221dfc63038097511f4f8552bccf68acbb2b2ef54bd9768d2fbcb2
softgpu-0.6.2024.40.zip: aae0f0cf25e9b315e159e3f942b2ebe4b199f9f98fc1cfdb37caad9d10ba46b6
```

</details>

<hr /><em>This discussion was created from the release <a href='https://github.com/JHRobotics/softgpu/releases/tag/v0.6.2024.40'>v0.6.2024.40-alpha3</a>.</em>

**Comment by DSN9990** [comment]
@JHRobotics When will you release the next version? I really need an very stable release of SoftGPU.

**Comment by JHRobotics** [comment]
@DSN9990 This is good question. I'm currently working on native DirectX driver, and this is probably only way to have DX7 and older apps and games stable.

I'm currently on DDI version 5, but some key features still missing (like mipmap or compressed textures support).

![dx-ddi-2024-11-19](https://github.com/user-attachments/assets/330ce478-ee78-435a-85b1-9380d788125f)

For the *BETA* I would like to have DDI 7 (feature set like GeForce MX 440), and this will be enough to run DX applications up to DX9, but without pixel shader support. This should be enough for near all Win9x games and there is still Wine for small group of games with PS requirement.

The next release should be in December. Probably :-)

**Comment by Nanami1911** [comment]
Is there any chance for the author to provide an update? 😭, it's been a long time

**Comment by DSN9990** [comment]
@JHRobotics When will you release the next version???? Why are you in another hiatus????

**Reply by DSN9990** [reply]
Please make pre-release builds for me to download, right?

**Reply by JHRobotics** [reply]
As you wish:

http://files.emulace.cz/distrib/softgpu-0.8.2025.50.iso
http://files.emulace.cz/distrib/softgpu-0.8.2025.50.zip
http://files.emulace.cz/distrib/softgpu-0.8.2025.50.iso.xz
